clients/src/main/java/org/apache/kafka/clients/producer/internals/ProducerBatch.java

```diff
@@ -76,13 +76,13 @@
     private String expiryErrorMessage;
     private boolean retry;
 
-    public ProducerBatch(TopicPartition tp, MemoryRecordsBuilder recordsBuilder, long now) {
-        this(tp, recordsBuilder, now, false);
+    public ProducerBatch(TopicPartition tp, MemoryRecordsBuilder recordsBuilder, long createTime) {
```

Comment:
The actual argument is `now`. However, I like the formal argument name to be `createTime` because it's an immutable value while constructing a batch. `now`, is by definition, changing.